src/Common/HashTable/HashTable.h

```diff
@@ -70,7 +70,7 @@ namespace ZeroTraits
 {
 
 template <typename T>
-bool check(const T x) { return x == T{}; }
+bool check(const T & x) { return x == T{}; }
```

Comment:
This change should not have any performance effects.

Reply:
@alexey-milovidov Could you please explain why? Thanks in advance.

Reply:
If the function body is in the same translation unit, the compiler can change the argument to pass-by-value.
Moreover, the function is inlined, and there will be no argument passing, neither by reference nor in registers.